### ISSUE TITLE
MWE fails at Line 196-198...

### ISSUE THREAD

**Issue by invertdna** [ramongallego/Rjags#1]
... with Error: `.x` must be a vector, not a function

The issue is the function model.round.nested(), which doesn't seem to do what you need it to do.

**Comment by ramongallego** [comment]
I can't install jags in this computer - I'll make it work at home

**Comment by invertdna** [comment]
No hurry, but for what it’s worth, it’s not a jags problem.

**Comment by ramongallego** [comment]
You are right - I think the problem is the data is not nested "enough" for
the function that does the rjags 10times - so I need to check wether the
function jags.for.presence() works with the input table as it is now - I
think I wrestled with this before
